lib/src/events/EventsRegistry.ts

```diff
@@ -65,4 +85,22 @@ export class EventsRegistry {
   public bindComponent(component: React.Component<any>, componentId?: string): EventSubscription {
     return this.componentEventsObserver.bindComponent(component, componentId);
   }
+
+  public notifyEvent(method: string, event?: NavEvent) {
+    if (this.listeners[method]) {
+      _.forEach(this.listeners[method], (callback) => {
+        callback(event);
```

Comment:
Seems to sometimes throw "callback is undefined" exceptions.  ... Looking into it.

Reply:
Should be good to go now.

Reply:
I'm still getting "callback is undefined". I think it has something to do with bound components. This worked for me:
```suggestion
      if (_.isFunction(callback)) {
        callback(event);
      }
```